src/main/java/com/google/api/codegen/transformer/ruby/RubyTestsTransformer.java

```diff
@@ -0,0 +1,58 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.transformer.ruby;
+
+import com.google.api.codegen.InterfaceView;
+import com.google.api.codegen.config.ApiConfig;
+import com.google.api.codegen.ruby.RubyGapicContext;
+import com.google.api.codegen.transformer.ModelToViewTransformer;
+import com.google.api.codegen.transformer.ModelTypeTable;
+import com.google.api.codegen.transformer.SurfaceNamer;
+import com.google.api.codegen.util.ruby.RubyTypeTable;
+import com.google.api.codegen.viewmodel.ViewModel;
+import com.google.api.tools.framework.model.Interface;
+import com.google.api.tools.framework.model.Model;
+import java.util.ArrayList;
+import java.util.List;
+
+/** A subclass of ModelToViewTransformer which translates model into API tests in Ruby. */
+public class RubyTestsTransformer implements ModelToViewTransformer {
+  // Template files
+  private static String UNIT_TEST_TEMPLATE_FILE = "ruby/test.snip";
+
+  @Override
+  public List<String> getTemplateFileNames() {
```

Comment:
You can use one-liner

return  ImmutableList.<String>of(UNIT_TEST_TEMPLATE_FILE);